native/packages/react-native-bpk-component-text-input/src/BpkTextInput-test.common.js

```diff
@@ -137,18 +137,6 @@ const commonTests = () => {
 
       expect(testRenderer.toJSON()).toMatchSnapshot();
     });
-
-    it('should ignore when placeholder is provided', () => {
```

Comment:
Is there any way we can test the placeholder support?

Reply:
I'm happy

Reply:
We could test that it's null by default (because the element isn't focused). Is there a way you know of to trigger focus in a test and then snapshot that?

Reply:
I've added a test for it being null when not focused (it's just the same test I removed, reinstated). Not sure about testing when focused though.